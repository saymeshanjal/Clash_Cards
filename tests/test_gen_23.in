11
4 218 1
14 176 3
4 308 1
6 138 4
2 361 3
10 30 2
2 6 2
10 302 2
2 297 5
16 79 5
16 80 5
