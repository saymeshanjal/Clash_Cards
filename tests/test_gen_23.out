3 3 0
1652 1780 0
3 3 0
1 14 5
0 2 0
56 61 135
0 1 1
76 77 99
0 2 0
4013 8857 0
4119 8751 0
